src/sdk/autodetectcompilers.cpp

```diff
@@ -180,7 +327,18 @@ void AutoDetectCompilers::OnUpdateUI(wxUpdateUIEvent& event)
 {
     wxListCtrl* list = XRCCTRL(*this, "lcCompilers", wxListCtrl);
     bool en = list->GetNextItem(-1, wxLIST_NEXT_ALL, wxLIST_STATE_SELECTED) != -1;
-    XRCCTRL(*this, "btnDefault", wxButton)->Enable(en);
+    XRCCTRL(*this, "btnSetDefaultCompiler", wxButton)->Enable(en);
 
+    wxString sDefaultCompiler = CompilerFactory::GetDefaultCompiler()->GetName();
+    for (auto itCL = std::begin(vCompilerList); itCL != std::end(vCompilerList); ++itCL)

```

Comment:
Range based for loop (without the auto) would be a lot better here.

Reply:
The code change to resolve this have been pushed.